chef_elasticsearch.yaml

```diff
@@ -44,12 +44,19 @@ Parameters:
   ElasticSearchVersion:
     Description: Version of ElasticSearch to use
     Type: String
-    Default: '5.6'
+    Default: '6.8'
```

Comment:
hey @gsreynolds , have you tested what would happen to an existing install at a previous version when the default changes?   At least in past versions of AWS ES, changing the version like this was a destructive change.

Reply:
AWS claim in-place upgrades from 5.x to 6.x are possible with caveats, but I'm testing the impact of that.

https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-version-migration.html

To be safe we can also leave the default ES version in main.yaml and marketplace.yaml as 5.6 and let customers opt in to deploying 6.x.

Reply:
Upgrade-in-place of the ElasticSearch domain is possible and works once that functionality is enabled in the template.

Manual upgrade check:
![Manual upgrade check](https://user-images.githubusercontent.com/1533562/82053993-278d3000-96b6-11ea-9cda-9667ce4eeff1.png)

CloudFormation triggered upgrade:
![CloudFormation triggered upgrade](https://user-images.githubusercontent.com/1533562/82055641-b8650b00-96b8-11ea-8cc0-3e89014822dd.png)

Successfully upgraded in place:
![Successfully upgraded in place](https://user-images.githubusercontent.com/1533562/82058411-a2594980-96bc-11ea-8437-9b7761f2615a.png)

Reply:
Awesome! nice work, and I really appreciated the comments with the doc links about version upgrades.  That's definitely improved since I last worked on this.

Reply:
Thanks, appreciate it. Worked surprisingly well when I tested it! Thanks for all your past work on this 👍